src/object/object_primitive.c

```diff
@@ -3568,7 +3568,7 @@ static int
 mr_data_readval_utime (OR_BUF * buf, DB_VALUE * value, TP_DOMAIN * domain, int size, bool copy, char *copy_buf,
 		       int copy_buf_len)
 {
-  DB_UTIME utm;
+  DB_UTIME utm = 0;
```

Comment:
This definition should be moved to the else branch of the if statement below because utm is used only in that branch.